apps/arweave/include/ar.hrl

```diff
@@ -759,6 +759,22 @@
 	signature_type = ?DEFAULT_KEY_TYPE
 }).
 
+-record(chunk_metadata, {
+	chunk_data_key = not_set :: not_set | not_found | binary(),
```

Comment:
not_found?

Reply:
Yeah that's a holdover from ar_repack. The data_path at least can be not found even when the chunks index metadata exists. 

So I need to distinguish not set vs not found for datapath at least. But I think not necessary for the other fields. 

I can remove the not found for those - but ok to leave for data path?

Reply:
Is it a case of a broken record when we miss an entry in chunk_data_db? What do we do with this chunk_metadata with data_path=not_found then?

Reply:
Yeah. If the chunks index data exists but the datapath does not then we invalidate the data record.

Reply:
I mostly needed a distinction between "not yet queried" and "queried but not found"

Reply:
Is this better?
https://github.com/ArweaveTeam/arweave/pull/765/commits/14f6c15f3ec5b9db1e2a1408f697978f33eac97c